src/Symfony/Bundle/FrameworkBundle/Command/DebugMessengerCommand.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Command;
```

Comment:
Should be moved to the component like the other Messenger commands.

Reply:
Done.